File: ccl-frontend/src/app/models/auth-response.ts
// Respuesta que devuelve el backend cuando haces login o register
export interface AuthResponse {
  token: string;       // Token JWT firmado
  username: string;    // Nombre de usuario autenticado
  role: string;        // Rol asignado (admin, user, etc.)
  expiresAt: string;   // Fecha de expiración del token
}

File: ccl-inventario/Ccl.Api/Dtos/AuthResponse.cs
namespace Ccl.Api.Dtos
{
    public class AuthResponse
    {
        public string Token { get; set; } = default!;
        public string Username { get; set; } = default!;
        public string Role { get; set; } = default!;
        public DateTime ExpiresAt { get; set; } 
    }
}
